include/word.h

```diff
@@ -19,15 +18,61 @@
 bool valid_word(dict_t *dict, char* word);
 
 /*
- * generate_suggestions - Returns a list of possible suggestions
+ * Helper function for generate_suggestions()
+ *
+ * word_check_cap - Checks if a word has a type of capitalization.
+ *
+ * Parameters:
+ *  - word: A string (word).
+ *
+ * Returns:
+ *  - -1: Error - Not a word.
```

Comment:
Please `#define` `ERROR`, `NOT_CAPITALIZED`, etc. What is this needed for though?

Reply:
The dictionary is composed of words that aren't capitalized, but some of the text is. Given this, when we input the word to generate suggestions, we need to use flags to recapitalize the returned outputs so that we can reinsert the corrected word without changes in capitalization.
The limit to this is that we can't currently correct inconsistent capitalization like `OhnoItsThEDeadLINe", but otherwise, this has been checked to be all right.
Specifically, this function is necessary for words_uppercase to work properly.
#define has been done.

Reply:
Oh gotcha. Neat, I hadn't even thought of that case.